internal/controller/policyserver_controller_test.go

```diff
@@ -660,6 +658,37 @@ var _ = Describe("PolicyServer controller", func() {
 			}).Should(Succeed())
 		})
 
+		It("should enable mTLS in the policy server deployment", func() {
+			policyServer := policiesv1.NewPolicyServerFactory().WithName(policyServerName).Build()
+			createPolicyServerAndWaitForItsService(ctx, policyServer)
+
+			deployment, err := getTestPolicyServerDeployment(ctx, policyServerName)
+			Expect(err).ToNot(HaveOccurred())
+
+			container := deployment.Spec.Template.Spec.Containers[0]
+
+			By("specifing the client ca certificate")
+			Expect(container.Env).To(ContainElement(MatchFields(IgnoreExtras, Fields{
+				"Name":  Equal("KUBEWARDEN_CLIENT_CA_FILE"),
+				"Value": Equal(fmt.Sprintf("%s,%s", constants.CARootCert, constants.ClientCACert)),
+			})))
+
+			Expect(deployment.Spec.Template.Spec.Volumes).To(ContainElement(MatchFields(IgnoreExtras, Fields{
```

Comment:
We are not checking the Kubewarden's root CA is mounted into the container. Are we?

Reply:
woops sorry! I have not commited the test